### DISCUSSION TITLE
logo中的logo.svg是否可以用文字代替？

### DISCUSSION THREAD

**Discussion by caopoy** [ppoffice/hexo-theme-icarus#898] [Q&A]
`_config.icarus.yml`文件下的logo项（就是指向/img/logo.svg的logo）可不可以填文字

就是可不可以用文字作为logo，而不是用图像

**Comment by ppoffice** [comment]
https://ppoffice.github.io/hexo-theme-icarus/Configuration/icarus%E7%94%A8%E6%88%B7%E6%8C%87%E5%8D%97-%E4%B8%BB%E9%A2%98%E9%85%8D%E7%BD%AE/#Logo
